dash.el

```diff
@@ -357,7 +357,7 @@ See also: `-each-indexed'."
        (nreverse ,r))))
 
 (defun -map-when (pred rep list)
-  "Return a new list where the elements in LIST that does not match the PRED function
+  "Return a new list where the elements in LIST that do not match the PRED function
```

Comment:
We should probably either drop `where` or `that`, or change the `where` to `with`.  This way it makes no sense to me.  There should also be a period at the end.